migration/insitution/app.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Simple script to post new Institutions given a file of institution names
+ *
+ * Requires authentication as an admin user.
+ *
+ * Sample Usage:
+ *   node app.js \
+ *     ./institutions.txt
+ *     `gcloud auth print-access-token` \
+ *     https://local.broadinstitute.org:27443 \
+ *
+ *
+ * EXAMPLE LINE IN FILE:
+ *  Polytechnique Fédérale de Lausanne
+ *
+ *  id, create user, create date, update user, and update date are automatically populated (even if given values they are ignored)
+ *  API call is in the form of:
+ *  POST $host/api/institutions -d {"name": "Polytechnique Fédérale de Lausanne"}
+ */
+
+const readline = require('readline');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/DataBiosphere/consent/pullRequest?prid=9547474)
```suggestion
const readline = require("readline");
```